pkgdown/_pkgdown.yml

```diff
@@ -12,6 +15,36 @@ template:
 
         gtag('config', 'G-PPRNFWFZBQ');
       </script>
+      <script async defer src="https://buttons.github.io/buttons.js"></script>
+
+url: https://appsilon.github.io/shiny.react/
 
 navbar:
-  type: inverse
+  bg: primary
+  structure:
+    left: [tutorial, reference, news]
+  components:
+    tutorial:
+      text: Tutorial
+      href: articles/shiny-react.html
+  right:
+    - icon: fa-github fa-lg
+      href: https://github.com/Appsilon/shiny.react
+    - icon: fa-twitter fa-lg
+      href: https://twitter.com/Appsilon
+
+home:
+  sidebar:
+    structure: [star, links, license, community, citation, authors, dev]
+    components:
+      star:
+        title: GitHub
+        text: |
+          <a class="github-button" href="https://github.com/appsilon/shiny.react" data-size="large" data-show-count="true" aria-label="Star appsilon/shiny.react on GitHub">Star</a>
```

Comment:
This seems to just display "Star" text linked to the repo main page. It doesn't display star count and clicking it doesn't add a star. I thought it might be different after pushing to GitHub pages, but it seems to be the same case for [shiny.router](https://appsilon.github.io/shiny.router/) page.

Reply:
Alright, the issue is due to my Adblock. The link still doesn't add a star though, just sends me to the main repo page. Is it supposed to be work like that?